_layouts/event.html

```diff
@@ -68,6 +71,9 @@
         {% if page.boxoffice_item_collection %}
         <a class="nav-item is-tab is-hidden-mobile" data-href="#tickets">Tickets</a>
         {% endif %}
+        {% if site.blog_feed_url %}
```

Comment:
Will this check on `site` lead to blog section showing up on every page? Should we change it to `page.blog_feed_url`

Reply:
Yes, I also recommend it to be a `page` item instead of `site` item